project.clj

```diff
@@ -3,7 +3,16 @@
   :url "http://example.com/FIXME"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.7.0"]]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [compojure "1.4.0"]
+                 [ring "1.4.0"]
+                 [bidi "1.25.0"]
+                 [clj-http "2.0.0"]
```

Comment:
compojure *and* bidi? I see you're using bidi, so maybe remove compojure from here?